src/components/Header/Header.tsx

```diff
@@ -1,9 +1,89 @@
-import React from 'react'
+import Button from '@mui/material/Button';
+import React from 'react';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import { Badge, IconButton, TextField } from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import styled from 'styled-components';
+import SearchOutlinedIcon from '@mui/icons-material/SearchOutlined';
+
+const Container = styled.div`
```

Comment:
What about exporting all styles to a seperate file in Header folder? Component would be cleaner